src/main/scala/nl/knaw/dans/easy/umd/Command.scala

```diff
@@ -59,8 +59,14 @@ object Command {
     new AutoDetectReader(new FileInputStream(file))
   }
 
-  private def requireUTF8(file: File, reader: AutoDetectReader) = Try {
-    require(reader.getCharset.toString == "UTF-8", s"encoding of $file must be UTF-8 but is ${reader.getCharset}")
+  val acceptedCharsets = Set("UTF-8", "ISO-8859-1")
```

Comment:
I know, but @lindareijnhoudt has to jump through some hoops to create a UTF-8 file with vi. We wanted to prevent that. From the first random test appears that exotic encodings are not safe, that made me impose the too strong requirement.

As [this test](https://github.com/DANS-KNAW-jp/easy-update-metadata-dataset/blob/a3dae76d600f6a67221022f2c8cd087424e63993/src/test/scala/nl/knaw/dans/easy/umd/CommandSpec.scala#L194) shows we had to accept this value. But it makes sense to accept `US-ASCII` too.